llvm/test/CodeGen/ARM/struct_byval.ll

```diff
@@ -67,33 +327,85 @@ declare i32 @e3(ptr nocapture byval(%struct.LargeStruct) align 16 %in) nounwind
 ; We can't do tail call since address of s is passed to the callee and part of
 ; s is in caller's local frame.
```

Comment:
Pretty sure this comment is still correct... we can't tail-call here.  r0 points below the bottom of the stack.

Reply:
I agree, I have removed the change to the test